R/get_gambl_metadata.R

```diff
@@ -458,34 +458,34 @@ get_gambl_metadata = function(seq_type_filter = "genome",
   
   # count NAs per row
   all_meta <- apply( all_meta, 1, function(x) sum(is.na(x)) ) %>% 
-    mutate(all_meta, na_count = .)
+    dplyr::mutate(all_meta, na_count = .)
   
   # take `seq_type_priority` lines
-  all_meta_res <- filter(all_meta, seq_type == seq_type_priority) %>% 
-    group_by(sample_id) %>% 
-    slice(which.min(na_count)) %>% 
-    ungroup
+  all_meta_res <- dplyr::filter(all_meta, seq_type == seq_type_priority) %>% 
+    dplyr::group_by(sample_id) %>% 
```

Comment:
Are all these `dplyr` prefixes called for? I know that `dplyr::filter` is needed since it has conflicting function name with some other package. But throughout the rest of GAMBLR we never tend to prefix other `dplyr` functions. I'm just curious to hear the reason behind this. I can understand, that it helps to some extent for a reviewer to see exactly what package any given function is coming from. Thanks!

Reply:
That is a good question. Out of the dplyr functions that I used here, `group_by` and `ungroup` seem to not **commonly** conflict with other functions. I prefixed them because other GAMBLR scripts do that and I thought I could miss that. Functions that cause conflicts: `mutate` (conflict between plyr and dplyr), `slice` (IRanges, dplyr), `filter` (stats, dplyr), `select` (MASS, dplyr), `arrange` (plyr, dplyr). There should be others --- these are the ones that I used here. Should I remove the prefix of any one of them?

Reply:
I don't think it matters too much. If anything, having them prefixed will highlight what package they come from. I was just curious about the rationale here since I don't think the majority of these functions are prefixed with `dplyr` throughout the rest of the code base (besides `filter`). Thanks!